### DISCUSSION TITLE
Error while librispeech conformer training

### DISCUSSION THREAD

**Discussion by awa292** [speechbrain/speechbrain#2343] [General]
Hey, while training  with Librispeech recipe on speechbrain 0.5.16, torch 2.1.2+cu121 I am facing this issue.

raise RuntimeError(f"Couldn't find appropriate backend to handle uri {uri} and format {format}.")
RuntimeError: Couldn't find appropriate backend to handle uri /home/sgeadmin/ext_hardisk/DroneDB/en/dataset/IN_SD/READ/2_pa_IN_SD_20230505/141260/pa_IN_SD_Male_141260_mobile_20230330165450.wav and format None.

Can somebody pointout what's the issue?

**Comment by awa292** [comment]
pip install PySoundFile, since torrchaudio does not report any backends available.

**Comment by TParcollet** [comment]
@mravanelli @Adel-Moumen another problem due to this change of backend policy of Torchaudio 2.1 we must find a way to fix this because it means that newcomers to SB will face an error as a first experience.
